pkg/ptr/ptr.go

```diff
@@ -37,3 +37,12 @@ func ToUint64(u *uint64) uint64 {
 	}
 	return *u
 }
+
```

Comment:
Maybe we should rename the package to "util", or "conv", since this function clearly doesn't involve any "ptr"s.

Reply:
done